Diary/Controller/ListViewController.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  Diary - ListViewController.swift
+//  Created by yagom. 
+//  Copyright © yagom. All rights reserved.
+// 
+
+import UIKit
+
+final class ListViewController: UIViewController {
+    private lazy var mainView = MainView.init(frame: view.bounds)
+    private var diaryArray: [Diary] = [] {
+        didSet {
+            mainView.tableView.reloadData()
+        }
+    }
+    
+    private lazy var plusButton: UIBarButtonItem = {
+        let plusButton = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(touchPlusButton))
+        
+        return plusButton
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setInitialView()
+    }
+    
+    private func setInitialView() {
+        self.view = mainView
+        mainView.tableView.dataSource = self
+        mainView.tableView.delegate = self
+        mainView.tableView.register(ListTableViewCell.self, forCellReuseIdentifier: "\(ListTableViewCell.self)")
+        configureNavigationBar()
+        convert(parsedData())
+    }
+    
+    private func configureNavigationBar() {
+        self.title = "일기장"
+        self.navigationItem.rightBarButtonItem = plusButton
+    }
+    
+    @objc private func touchPlusButton() {
+        let diaryViewController = DiaryViewController(diary: nil, type: .add)
+        
+        self.navigationController?.pushViewController(diaryViewController, animated: true)
+    }
+    
+    private func parsedData() -> [DiaryData]? {
+        guard let asset = NSDataAsset(name: "sample") else {
+            return nil
+        }
+        
+        let jsonDecoder = JSONDecoder()
+        
+        guard let sample = try? jsonDecoder.decode([DiaryData].self, from: asset.data) else {
+            return nil
+        }
+        
+        return sample
+    }
+    
+    private func convert(_ data: [DiaryData]?) {
+        guard let diaryDatas = data else {
+            return
+        }
+        
+        var array: [Diary] = []
+        
+        for diaryData in diaryDatas {
+            guard let doubleDate = diaryData.createdAt else {
+                return
+            }
+            
+            let date = Date(timeIntervalSince1970: doubleDate)
+            let stringDate = DateConverter.changeDateFormat(date)
```

Comment:
DateConverter를 따로 작성하지 않고 Date의 extension에서 포맷화된 String 데이터를 반환하면 바로 date 변수를 활용할 수 있을 것 같은데 어떠신가요? 
```suggestion
            guard let doubleDate = diaryData.createdAt else {
                return
            }
            
            let date = Date(timeIntervalSince1970: doubleDate)
            let stringDate = date.dateToKoreanString // "yyyy년 mm월 dd일"을 반환하는 연산 프로퍼티
```